File: 136-single-number/single-number.cpp
class Solution {
public:
    int singleNumber(vector<int>& nums) {
        unordered_map<int,int>st;
        for(auto it :nums)
        {
            st[it]++;
        }
        for(int i=0;i<nums.size();i++)
        {
            if(st[nums[i]]==1)
            {
                return nums[i];
            }
        }
        return -1;
    }
};

File: 136-single-number/single-number.java
class Solution {
    public int singleNumber(int[] nums) {
        HashMap<Integer,Integer>mp=new HashMap<>();
        for(int x:nums){
            mp.put(x,mp.getOrDefault(x,0)+1);
        }
        for(int i=0;i<nums.length;i++)
        {
            if(mp.get(nums[i])==1)
            {
                return nums[i];
            }
        }
        return -1;

    }
}